compiled-templates/about.md

```diff
@@ -1,73 +1,52 @@
-on new feature ideas because we have never had a good mechanism for iterating and improving over time.
+These decisions aren't just academic: implementing a new feature in Firefox can take thousands of hours of design and engineering time. We test with prototypes, but don't always get the kind of real-world feedback that helps us understand how people will use the feature and how well it really performs for them.
 
-Test Pilot is more than just the web app you're currently looking at. It's a *research, design, and development process* aimed at making feature exploration and development less costly, 
-while allowing us to play with more radical ideas for Firefox. The process consists of four parts.
+Test Pilot is more than just the web app you're currently looking at. It's a *research, design, and development process* aimed at making feature development a smoother, more transparent process, while allowing us to play with — and share — more radical ideas for Firefox. The process consists of four phases.
 
-#### *Phase One:* Low Fidelity Validation
+#### *Phase One:* Prototype and Validate
 
-In *Phase One*, The Test Pilot Team solicits ideas for new features from across Mozilla (soon we'll be asking our users to contribute as well).
+In *Phase One*, The Test Pilot team evaluates lots of promising ideas. First, we do research to better understand the problems we want to solve and to uncover potential solutions. Depending on the problem, our experiment may be an entirely new feature or it could be a design that helps us gather information to improve existing features.
 
-When we hear a promising idea, we conduct a bunch of research to define both the problem we want to solve, and 
-the possible shapes that our experimental solutions might take. We look at existing strategies in the market for solving similar problems, and we spend a ton of time sketching and
-tinkering with simple prototypes. Sometimes these prototypes are doodles on paper, sometimes we mock things up in PhotoShop, and sometimes we explore in code. The name
-of the game in this phase is to learn quickly and cheaply. If an idea doesn't make it out of *Phase One*, that's just fine. We've saved ourselves a lot of headache down the road
-and learned something to boot. If an experiment graduates to *Phase Two*, we start to get a little more serious. 
+Then, we sketch and tinker with simple prototypes. Sometimes these prototypes are doodles on paper, sometimes we mock things up in Photoshop, and sometimes we explore in code. We test these prototypes in front of real Firefox users help us validate and refine our ideas. The name of the game in this phase is to learn quickly. If an idea doesn't make it out of *Phase One*, that's just fine. We've saved ourselves a lot of headaches down the road and learned something to boot. If an experiment graduates to *Phase Two*, we start to get a little more serious. 
```

Comment:
"We test these prototypes in front of real Firefox users help us validate and refine our ideas." is missing a word somewhere